assets/tests/hooks/useVehiclesForRunIds.test.ts

```diff
@@ -0,0 +1,144 @@
+import { renderHook } from "@testing-library/react-hooks"
+import useVehicleForRunIds from "../../src/hooks/useVehiclesForRunIds"
+import { VehicleData } from "../../src/models/vehicleData"
+import { dateFromEpochSeconds } from "../../src/util/dateTime"
+import { makeMockChannel, makeMockSocket } from "../testHelpers/socketHelpers"
+
+const vehicleData: VehicleData = {
```

Comment:
we've really got to start moving towards factories, there's no reason either of us should spend time writing the literally 60 fields this object needs by hand.

Reply:
not to mention the noise and bulk it adds to PRs.